app/views/users/_account_set_password.html.erb

```diff
@@ -1,27 +1,29 @@
-  <div class="crayons-card crayons-card--content-rows">
-    <header>
-      <h2 class="crayons-subtitle-1">
-        Set new password
-      </h2>
-      <p class="color-base-70">
-        As an alternative to signing in via linked social accounts, you may create a password.
-
-        If you signed up with a social media account, please <a href="<%= new_user_password_path %>">reset the password</a> for your primary email address (<%= current_user.email %>) first.
-      </p>
-    </header>
-    <%= form_with(url: user_update_password_path, data: { testid: "update-password-form" }) do |f| %>
-      <div class="crayons-field mb-4">
-        <%= f.label :current_password, class: "crayons-field__label" %>
-        <%= f.password_field :current_password, class: "crayons-textfield", autocomplete: "off", required: true %>
-      </div>
-      <div class="crayons-field mb-4">
-        <%= f.label :password, class: "crayons-field__label" %>
-        <%= f.password_field :password, class: "crayons-textfield", autocomplete: "off", required: true %>
-      </div>
-      <div class="crayons-field mb-4">
-        <%= f.label :password_confirmation, "Confirm new password", class: "crayons-field__label" %>
-        <%= f.password_field :password_confirmation, autocomplete: "off", class: "crayons-textfield", required: true %>
-      </div>
-      <button class="crayons-btn crayon-btn--secondary" type="submit">Set New Password</button>
-   <% end %>
+<div class="crayons-card crayons-card--content-rows">
+  <header>
+    <h2 class="crayons-subtitle-1">
+      Set new password
+    </h2>
+    <p class="color-base-70">
+    If your account was created using social account authentication, you may prefer to add an email
+    log in. If you signed up with a social media account, <a href="<%= new_user_password_path %>">
+      please reset the password</a> for your primary email address (<%= current_user.email %>) in
+    order to enable this. Please note that email log in is in addition to social log in rather than
```

Comment:
I know that you just copied the changes from the issue, but I think we need to use the noun `login` within the context of this sentence and not the verb `log in`.
```suggestion
    order to enable this. Please note that email login is in addition to social login rather than
```

Reply:
Only realized this after I approved @citizen428.